### ISSUE TITLE
Docker images are built without version details

### ISSUE THREAD

**Issue by 06kellyjac** [controlplaneio/kubesec#121]
```bash
$ docker run --rm -i docker.io/kubesec/kubesec:v2 version
version unknown
git commit unknown
build date unknown
```

**Comment by 06kellyjac** [comment]
Im going to fix this as part of #250

**Comment by JOSHUAJEBARAJ** [comment]
I think both of  are different issue because the go-releaser only changes the binary version that is present in the release page I won't change the binary present inside the container  [here](https://github.com/controlplaneio/kubesec/blob/master/Dockerfile#L7)

**Comment by 06kellyjac** [comment]
Yes I'm aware but I'm updating the version command so I'm fixing both separate issues at the same time.

The version will now list the version, full commit, source date epoch in RFC 3339 format, and unix epoch format

I'll either try plug those details into the docker build action or utilize goreleaser within the containers

**Comment by marcio-pessoa** [comment]
This issue is still happening:

```sh
$ docker run --rm kubesec/kubesec version
Unable to find image 'kubesec/kubesec:latest' locally
latest: Pulling from kubesec/kubesec
a0d0a0d46f8b: Pull complete 
225a6626967c: Pull complete 
50cb987b6fe2: Pull complete 
404740fbf2d0: Pull complete 
66230eef9706: Pull complete 
7cd616b8ef56: Pull complete 
Digest: sha256:1085be3c2494be9d8dc4884e7ceaf7a79fca650f02d1e36d6b1ac317e0495cb3
Status: Downloaded newer image for kubesec/kubesec:latest
version unknown
git commit unknown
build date unknown
```

**Comment by MarioUhrik** [comment]
Hi @06kellyjac, if you're interested, I can try to open a PR where I fix this issue by:
- supplying git tag/commit/date information from GitHub Actions into the Docker Build GHA, [e.g. like this](https://dev.to/jpoehnelt/environment-variables-in-github-docker-build-push-action-23pj)
- editing the Dockerfiles so the env variables from the previous step are passed into `ldflags` argument of `go build`, [e.g. like this (Dockerfile in the question)](https://github.com/ko-build/ko/issues/167)
- as a bonus, fix the harmless path typo at https://github.com/controlplaneio/kubesec/blob/master/Dockerfile.scratch#L13

**Comment by 06kellyjac** [comment]
Thanks for the offer but there are a couple other tweaks I want to make to the Dockerfiles so ill do that in 1 pr
